vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/component/ExperimentInfoComponent.java

```diff
@@ -0,0 +1,168 @@
+import java.util.List;
+import java.util.Objects;
+import life.qbic.datamanager.views.general.Card;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Analyte;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Species;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Specimen;
+
+/**
+ * <b>Experiment Info Component</b>
+ *
+ * <p>A component that nicely renders experiment-related general information</p>
+ *
+ * @since 1.0.0
+ */
+public class ExperimentInfoComponent extends Card {
+
+  private boolean controlsEnabled = true;
+  @Serial
+  private static final long serialVersionUID = -4790635833822470484L;
+  public static final String TITLE = "title";
+  private final Collection<Species> species;
+  private final Collection<Specimen> specimen;
+  private final Collection<Analyte> analytes;
+  private final Div controls = new Div();
+  private final MenuBar menuBar;
+  private final List<ComponentEventListener<ExperimentInfoEditEvent>> editListeners = new ArrayList<>();
+
+  private ExperimentInfoComponent(Collection<Species> species, Collection<Specimen> specimen,
+      Collection<Analyte> analytes) {
+    super();
+    this.species = species;
+    this.specimen = specimen;
+    this.analytes = analytes;
+    this.menuBar = createMenuBar();
+    layoutComponent();
+  }
+
+  private MenuBar createMenuBar() {
+    MenuBar menu = new MenuBar();
+    menu.addThemeVariants(MenuBarVariant.LUMO_TERTIARY);
+    MenuItem menuItem = menu.addItem("•••");
+    SubMenu subMenu = menuItem.getSubMenu();
+    subMenu.addItem("Edit", event -> {
+      fireEditEvent();
+    });
+    return menu;
+  }
+
+  private void layoutComponent() {
+    Div cardHeader = new Div();
+    cardHeader.addClassName("header");
+
+    controls.addClassName("controls");
+    controls.add(menuBar);
+
+    cardHeader.add(new Span("Fact Sheet"));
+    cardHeader.add(controls);
+    this.add(cardHeader);
+
+    Div content = new Div();
+    Span span = new Span("Species");
+    span.addClassName(TITLE);
+    var speciesList = speciesList();
+    content.add(span, speciesList);
+
+    Span specimenTitle = new Span("Specimen");
+    specimenTitle.addClassName(TITLE);
+    var specimenList = specimenList();
+    content.add(specimenTitle, specimenList);
+
+    Span analytesTitle = new Span("Analytes");
+    analytesTitle.addClassName(TITLE);
+    var analytesList = analytesList();
+    content.add(analytesTitle, analytesList);
+    content.addClassName("content");
+
+    add(content);
+    addClassName("experiment-info-card");
+  }
+
+  private void fireEditEvent() {
+    var event = new ExperimentInfoEditEvent(this, true);
+    editListeners.forEach(listener -> listener.onComponentEvent(event));
+  }
+
+  private UnorderedList speciesList() {
+    return new UnorderedList(
+        this.species.stream().map(thisSpecies -> new ListItem(thisSpecies.value())).toList()
+            .toArray(ListItem[]::new));
+  }
```

Comment:
why not call `toArray` directly instead of `toList().toArray`?

Reply:
great, was not even suggested by IntelliJ 💯